QUEST_LV_0200.tsv

```diff
@@ -384,16 +384,16 @@ QUEST_LV_0200_20150317_000383	There is a monocle that helps you identify preciou
 QUEST_LV_0200_20150317_000384	The scroll Bilhelmina gave us is awesome!{nl}I didn't hear a thing even from right beside.
 QUEST_LV_0200_20150317_000385	Shall we go now?
 QUEST_LV_0200_20150317_000386	I'm a wizard who can't use any magic. I feel pathetic.
-QUEST_LV_0200_20150317_000387	Antares.. Used to be a great wizard back in the day.
+QUEST_LV_0200_20150317_000387	Antares used to be a great wizard back in the day.
```

Comment:
I think the previous line is trying to create a pause after Antares. Try adding an ellipse … so we maintain that. :)